### PR TITLE
[Enhencement](schema_scanner) Optimize the performance of reading information schema tables

### PR DESCRIPTION
**Pull request by Cai-Yao**
# Proposed changes

Issue Number: close #xxx

## Problem summary

1. batch fill block
2. batch call rpc from FE to get table desc

For 34w colunms
```
SELECT COUNT( * ) FROM information_schema.columns;
```
time: 10.3s --> 0.4s

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Cai-Yao** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 33.49 seconds
 stream load tsv:          483 seconds loaded 74807831229 Bytes, about 147 MB/s
 stream load json:         39 seconds loaded 2358488459 Bytes, about 57 MB/s
 stream load orc:          68 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230302175918_clickbench_pr_107584.html

**Comment by yiguolei** [comment]
./run p0

File: regression-test/suites/query_p0/system/test_query_sys_tables_with_lots_of_tables.groovy [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                "storage_format" = "V2"
+                )
+            """
+        }
+    }
+
+    for (int i = 1; i <= 10; i++) {
+        def dbName = dbPrefix + i.toString()
+        sql "USE information_schema"
+        qt_sql "SELECT COUNT(*) FROM `columns` WHERE TABLE_SCHEMA='${dbName}'"
```
Comment (morningman) [inline regression-test/suites/query_p0/system/test_query_sys_tables_with_lots_of_tables.groovy:80]: better to add more cases to cover all schema tables

Comment (morningman) [reply regression-test/suites/query_p0/system/test_query_sys_tables_with_lots_of_tables.groovy:80]: Or you can move this cases to the previous case file

Comment (Cai-Yao) [reply regression-test/suites/query_p0/system/test_query_sys_tables_with_lots_of_tables.groovy:80]: > Or you can move this cases to the previous case file

Ok, I think it's better that way too.

**Comment by yiguolei** [comment]
./run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
./run p0

**Comment by Cai-Yao** [comment]
run p0

**Comment by yiguolei** [comment]
./run buildall

**Comment by Cai-Yao** [comment]
run p0

**Comment by Cai-Yao** [comment]
./run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**